examples/complete/variables.tf

```diff
@@ -1 +1,62 @@
-#(empty)
+variable "name" {
+  description = "(Required) This is the name of the hosted zone."
+  type        = string
+  default     = "boldlink-complete-example.com"
+}
+
+variable "cidr_block" {
+  type        = string
+  description = "VPC CIDR"
+  default     = "10.1.0.0/16"
+}
+
+variable "enable_public_subnets" {
+  type        = bool
+  description = "Whether to enable public subnets"
+  default     = true
+}
+
+variable "map_public_ip_on_launch" {
+  type        = bool
+  description = "Whether assign public IPs by default to instances launched on subnet"
+  default     = true
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "Tags to apply to the created resources"
+  default = {
+    Environment        = "examples"
+    "user::CostCenter" = "terraform-registry"
+    Department         = "DevOps"
+    Project            = "Examples"
+    Owner              = "Boldlink"
```

Comment:
See my comment below for this about Owner. Probably we set this to `hugo.almeida`